### PR TITLE
Added README to integration

### PR DESCRIPTION
**Pull request by AdrianaRoseDiaz**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/content/issues/17420
fixes: https://github.com/demisto/etc/issues/45000

## Description
Added README as per the issue.

## Screenshots
NA

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by yucohen** [comment]
@AdrianaRoseDiaz Thanks for your contribution! looks great!
Please sign the form mentioned in [this comment](https://github.com/demisto/content/pull/18186#issuecomment-1071017827) and then we can merge

**Comment by bakatzir** [comment]
@yucohen please close all of the related issues when the internal PR is merged https://github.com/demisto/etc/issues/45000

File: Packs/APIVoid/Integrations/APIVoid/README.md [hunk 1]
```diff
@@ -0,0 +1,816 @@
+APIVoid wraps up a number of services such as ipvoid & urlvoid
+This integration was integrated and tested with version 01 of APIVoid
```
Comment (bakatzir) [inline Packs/APIVoid/Integrations/APIVoid/README.md:2]: is there an external link to the `01` we can reference?

File: Packs/APIVoid/Integrations/APIVoid/README.md [hunk 2]
```diff
@@ -0,0 +1,816 @@
...
+
+## Configure APIVoid on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for APIVoid.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | API KEY |  | True |
```
Comment (bakatzir) [inline Packs/APIVoid/Integrations/APIVoid/README.md:12]: how do you generate an API KEY? we usually add this info (example) in: https://github.com/demisto/content/blob/master/Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2_description.md

File: Packs/APIVoid/Integrations/APIVoid/README.md [hunk 3]
```diff
@@ -0,0 +1,816 @@
...
+    | Good Reputation (Percentage) | If the percentage of detections is BELOW this value, the indicator is considered GOOD | True |
```
Comment (bakatzir) [inline Packs/APIVoid/Integrations/APIVoid/README.md:13]: Is there an external reference from APIvoid about the `Percentage`? I suggest adding it to the `_description.md` file as well.

File: Packs/APIVoid/ReleaseNotes/1_0_12.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+##### APIVoid
+- Updated README in the integration.
```
Comment (bakatzir) [inline Packs/APIVoid/ReleaseNotes/1_0_12.md:2]: As the README resides in xsoar.pan.dev and not within XSOAR, we prefer adding this generic ReleaseNotes note usually.
```suggestion
- Documentation and metadata improvements.
```

File: Packs/APIVoid/Integrations/APIVoid/README.md (hunk 1 above)
Comment (AdrianaRoseDiaz) [reply Packs/APIVoid/Integrations/APIVoid/README.md:2]: There is none, I wrote version 1, since at first I got an error for having XX.

Comment (bakatzir) [inline Packs/APIVoid/Integrations/APIVoid/README.md:2]:
```suggestion
```

File: Packs/APIVoid/Integrations/APIVoid/README.md (hunk 3 above)
Comment (bakatzir) [reply Packs/APIVoid/Integrations/APIVoid/README.md:12]: ^^ @AdrianaRoseDiaz a slight nudge :)

Comment (AdrianaRoseDiaz) [reply Packs/APIVoid/Integrations/APIVoid/README.md:12]: The percentage is up for the user to decide, if the percentage of detections is bellow a value (for example, if less than 10% of users declared it as suspicious), the indicator is considered GOOD. So in essence, the score is not static to one thing, it depends on user scores.

File: Packs/APIVoid/Integrations/APIVoid/README.md [hunk 5]
```diff
@@ -0,0 +1,815 @@
...
+    | Suspicious Reputation (Percentage) | If the percentage of detections is ABOVE this value, the indicator is considered SUSPICIOUS | True |
+    | Bad Reputation (Percentage) | If the percentage of detections is ABOVE this value, the indicator is considered BAD | True |
```
Comment (bakatzir) [inline Packs/APIVoid/Integrations/APIVoid/README.md:14]:
```suggestion
    | Malicious Reputation (Percentage) | If the percentage of detections is ABOVE this value, the indicator is considered Malicious | True |
```

**Review by bakatzir — APPROVED** [review]
I've editted the readme to use the benign/suspicious/malicious terminology.

File: Packs/APIVoid/Integrations/APIVoid/README.md [hunk 6]
```diff
@@ -0,0 +1,815 @@
...
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for APIVoid.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | API KEY |  | True |
+    | Benign Reputation (Percentage) | If the percentage of detections is BELOW this value, the indicator is considered Benign | True |
+    | Suspicious Reputation (Percentage) | If the percentage of detections is ABOVE this value, the indicator is considered SUSPICIOUS | True |
```
Comment (bakatzir) [inline Packs/APIVoid/Integrations/APIVoid/README.md:13]:
```suggestion
    | Suspicious Reputation (Percentage) | If the percentage of detections is ABOVE this value, the indicator is considered Suspicious | True |
```

File: Packs/APIVoid/Integrations/APIVoid/README.md [hunk 7]
```diff
@@ -0,0 +1,815 @@
...
+2. Search for APIVoid.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | API KEY |  | True |
+    | Benign Reputation (Percentage) | If the percentage of detections is BELOW this value, the indicator is considered Benign | True |
+    | Suspicious Reputation (Percentage) | If the percentage of detections is ABOVE this value, the indicator is considered Suspicious | True |
+    | Malicious Reputation (Percentage) | If the percentage of detections is ABOVE this value, the indicator is considered Malicious | True |
+    | Malicious | Consider the indicator malicious if either "suspicious" or "bad" | True |
```
Comment (bakatzir) [inline Packs/APIVoid/Integrations/APIVoid/README.md:15]:
```suggestion
    | Malicious | Consider the indicator malicious if either Suspicious or Malicious | True |
```

**Pull request merged**